app/src/main/java/de/danoeh/antennapod/adapter/SubscriptionsRecyclerAdapter.java

```diff
@@ -220,6 +237,24 @@ public void bind(NavDrawerData.DrawerItem drawerItem) {
                         .load();
             }
         }
+
+        private void updateLayoutParams(TextView feedTitle, int textColor) {
+            lastParams = feedTitle.getLayoutParams();
```

Comment:
This is a bit weird. You copy the layout params of random views into the variable and then apply the copied params to all other views. I think it might be better to use ViewTypes (see `getItemViewType`) that then set the layout params one single time when the views are created.